### DISCUSSION TITLE
Welcome to Pruned-OpenVINO-YOLO Discussions!

### DISCUSSION THREAD

**Discussion by TNTWEN** [TNTWEN/Pruned-OpenVINO-YOLO#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by pieterSteenekamp** [comment]
Thanks for the good work,
I can successfully convert the yolo v4 to open vino format, but only for FP32 format.
I want to use it on my raspberry pi with a NCS2 stick, and for that I need it to be FP16.
Is it possible to get the yolo v4 in  open vino TF16 format? 
Can anybody help me to do that please, I'll be very grateful
Pieter Steenekamp

**Reply by TNTWEN** [reply]
@pieterSteenekamp

FP32 ,FP16
The default precision is FP32,if you want to use FP16 ,add --data_type FP16 when you run mo.py

**Comment by pieterSteenekamp** [comment]
Thank you very much. I tried it and it works! I'm happy now.

Regards
Pieter Steenekamp
